parity/secretstore.rs

```diff
@@ -165,6 +161,11 @@ mod server {
 				None => return Err("self secret is required when using secretstore".into()),
 			};
 
+			info!("Starting SecretStore node: {}", White.bold().paint(format!("{:?}", self_secret.public())));
```

Comment:
How about being explicit here and say `Starting SecretStore node with pub key: {}`?

Reply:
I'd like to keep it as short as possible (this line is already too long) - not sure if that gives any additional information :) But could replace, if you insist it is better :)